cypress/support/child-commands.ts

```diff
@@ -30,10 +21,6 @@ declare namespace Cypress {
   }
 }
 
-Cypress.Commands.add('shouldBeVisible', {prevSubject: 'element'}, (subject) => {
```

Comment:
So these commands are confusing me.

It looks like you've replaced all usage of `shouldBeVisible` with the native `.should('be.visible')`, but then replaced `.should('have.attr', 'aria-expanded', 'true');` with `shouldHaveAttribute` in other places.

It looks like you pulled out the one command that was being used, and then started using another one for a different purpose.

I'm also not entirely convinced that 

`.shouldHaveAttribute('aria-expanded', 'true')`

is much, if not any, improvement over the native

`.should('have.attr', 'aria-expanded', 'true');`

Maaaaaybe lets' just delete this file and commands entirely?